src/foam/u2/borders/MarginBorder.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'MarginBorder',
+  extends: 'foam.u2.Element',
+
+  documentation: `
+    A border which adds margin to all sides except the bottom.
+  `,
+
+  properties: [
+    {
+      name: 'margin',
+      class: 'Int',
+      value: 24
+    },
+    {
+      name: 'marginUnit',
+      class: 'String',
+      value: 'px'
+    },
+
+    {
```

Comment:
Why not just let the user type in whatever they want. They they can specify just one margin or 2 or 4 margins, like with regular CSS?

Reply:
If I understand, you're suggesting a `.style({ margin: '24px' })` can do the same thing, right?

I was thinking if margin size is added to themes then this model could be modified to reflect that.

Reply:
Just let them specify 'margin' as a string, rather than breaking into unit an int and a unit. Then the user can also specify: 'auto', 'inherit', or something like: '25px 50px 75px 100px;' or '75px 100px;'
It's just more flexible if you try to be less helpful.